packages/vscode-extension/src/commonlib/codeFlowLogin.ts

```diff
@@ -216,6 +222,55 @@ export class CodeFlowLogin {
     return accessToken;
   }
 
+  async loginInCodeSpace(scopes: Array<string>): Promise<string> {
+    let callbackUri: Uri = await env.asExternalUri(
+      Uri.parse(`${env.uriScheme}://TeamsDevApp.ms-teams-vscode-extension/auth-complete`)
+    );
```

Comment:
Use `extensionID` constant instead since you've defined it in `common/constant.ts`.